vector/src/main/java/im/vector/app/features/userdirectory/UserListController.kt

```diff
@@ -141,25 +141,25 @@ class UserListController @Inject constructor(private val session: Session,
             is Fail    -> {
                 when (matchingEmail.error) {
                     is IdentityServiceError.UserConsentNotProvided     -> {
-                        genericPillItem {
-                            id("consent_not_given")
-                            text(
-                                    span {
-                                        span {
-                                            text = host.stringProvider.getString(R.string.settings_discovery_consent_notice_off_2)
-                                        }
-                                        +"\n"
-                                        span {
-                                            text = host.stringProvider.getString(R.string.settings_discovery_consent_action_give_consent)
-                                            textStyle = "bold"
-                                            textColor = host.colorProvider.getColorFromAttribute(R.attr.colorPrimary)
-                                        }
-                                    }
-                            )
-                            itemClickAction {
-                                host.callback?.giveIdentityServerConsent()
-                            }
-                        }
+//                        genericPillItem {
+//                            id("consent_not_given")
+//                            text(
+//                                    span {
+//                                        span {
+//                                            text = host.stringProvider.getString(R.string.settings_discovery_consent_notice_off_2)
+//                                        }
+//                                        +"\n"
+//                                        span {
+//                                            text = host.stringProvider.getString(R.string.settings_discovery_consent_action_give_consent)
+//                                            textStyle = "bold"
+//                                            textColor = host.colorProvider.getColorFromAttribute(R.attr.colorPrimary)
+//                                        }
+//                                    }
+//                            )
+//                            itemClickAction {
+//                                host.callback?.giveIdentityServerConsent()
+//                            }
+//                        }
```

Comment:
Can you confirm this is not required to comment this code? (because consent is supposed to be always true)

Reply:
Indeed, I reverted this change